### PR TITLE
change name convention

### PR DESCRIPTION
**Pull request by zaliqarosli**
hi @sanuann, please check that this is how you want the name convention to be. thanks

### CODE REVIEW

File: activities/MedicalHistoryLORIS/MedicalHistoryLORIS_schema [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
     "@context": [ "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/contexts/generic",
-        "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/MedicalHistory-LORIS/medicalhistory_context"
+        "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/MedicalHistoryLORIS/MedicalHistoryLORIS_context"
     ],
     "@type": "reproschema:Activity",
-    "@id": "medicalhistory_schema",
+    "@id": "MedicalHistoryLORIS_schema",
```
Comment (sanuann) [inline activities/MedicalHistoryLORIS/MedicalHistoryLORIS_schema]: please remove `_schema` at the end of '@id' string i.e. MedicalHistoryLORIS (same as directory name). 

Please make the corrections for '@id' for others too.

Comment (zaliqarosli) [reply activities/MedicalHistoryLORIS/MedicalHistoryLORIS_schema]: @sanuann shouldn't the @id match the IRI of the schema?

Comment (zaliqarosli) [reply activities/MedicalHistoryLORIS/MedicalHistoryLORIS_schema]: if i remove `_schema`, i believe the `@id` would expand out to `https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/MedicalHistory-LORIS/MedicalHistoryLORIS` which wouldn't be an existing IRI

Comment (sanuann) [reply activities/MedicalHistoryLORIS/MedicalHistoryLORIS_schema]: ok makes sense.

**Pull request merged**